packages/mobile/src/setupE2eEnv.e2e.ts

```diff
@@ -0,0 +1,26 @@
+import Config from 'react-native-config'
+// @ts-ignore
+import { TimerContext } from 'victory-core'
+// @ts-ignore
+import { TimerContext as TimerContextES } from 'victory-core/es'
+
+// Inject custom e2e config
+Config.IS_E2E = true
```

Comment:
Why inject this in here instead of setting `IS_E2E=true` in ` .env.test`? I thought I added it there already, I think that was my mistake

Reply:
There was a script adding this variable to the env file, but it was removed recently and IS_E2E wasn't set anywhere anymore.
I though about adding it back to the env file, but then I found that it was more flexible to have it injected this way as it allows using any existing env file without modifying it.